cmd/storj-sim/network.go

```diff
@@ -414,9 +414,19 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			process.WaitForStart(satellite)
 		}
 
+		// TODO: reuse for satellite and storagenode operator
+		// TODO: find source file, to set static path
+		_, filename, _, ok := runtime.Caller(0)
+		if !ok {
+			return nil, errs.Combine(processes.Close(), errs.New("no caller information"))
+		}
+		storjRoot := strings.TrimSuffix(filename, "/cmd/storj-sim/network.go")
```

Comment:
Could we put these files inline/ dont require external paths?

Reply:
that's for running with `storj-sim only`, but, yeah, we won't need if we bundle assets with the binary

Reply:
But the same needs to be applied in the storagenode binary directly, right?

Reply:
as `storj-sim` using pre-built binaries, we won't these lines if we bundle assets directly to the binary as there won't be any path for resources at all. This code originally was for `satellite` running with `storj-sim`, to bypass setting config values for external resources to be able to run `storj-sim`(which assumes in some way your building from sources)